.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+Fixes #Issue_Number
+
+Please Add Screenshots If any UI changes.
```

Comment:
Change it to - `Please Add Screenshots If there are any UI changes`

Reply:
@luckyman20 done.